contents/forms/checkbox/checkboxes/index.tsx

```diff
@@ -0,0 +1,14 @@
+import { Checkbox, CheckboxGroup } from "@yamada-ui/react"
+import type { FC } from "react"
+
+const Checkboxes: FC = () => {
+  return (
+    <CheckboxGroup>
+      <Checkbox>Yamada UI</Checkbox>
+      <Checkbox>React component library</Checkbox>
+      <Checkbox>Bringing your ideas to life</Checkbox>
+    </CheckboxGroup>
+  )
+}
+
+export default Checkboxes
```

Comment:
Please change `Checkboxes` to `GroupedCheckboxes` to be more clearer.

```suggestion
const GroupedCheckboxes: FC = () => {
  return (
    <CheckboxGroup>
      <Checkbox>Yamada UI</Checkbox>
      <Checkbox>React component library</Checkbox>
      <Checkbox>Bringing your ideas to life</Checkbox>
    </CheckboxGroup>
  )
}

export default GroupedCheckboxes
```

Reply:
I fixed.